pkg/operator/ceph/cluster/mon/config.go

```diff
@@ -39,8 +39,8 @@ import (
 )
 
 const (
-	// All mons share the same keyring
-	keyringStoreName = "rook-ceph-mons"
+	// KeyringStoreName is the keyringname shared by all mons
```

Comment:
Why does `crash` need the monitor keyring? I thought it needed the admin key.

Reply:
This key has both, I should investigate how to get the admin key only.

Reply:
You can investigate the operator/ceph/cluster/mgr code to see how it uses `keyring.VolumeMount().Admin()` to mount the admin keyring.